cmd/lncli/cmd_invoice.go

```diff
@@ -106,15 +182,19 @@ func addInvoice(ctx *cli.Context) error {
 		amt, err = strconv.ParseInt(args.First(), 10, 64)
 		args = args.Tail()
 		if err != nil {
-			return fmt.Errorf("unable to decode amt argument: %w",
-				err)
+			return fmt.Errorf(
+				"unable to decode amt argument: %w", err,
+			)
```

Comment:
was good it was :) - logs and return errors dont have the same formatting rules as other code blocks